site/generate.sh

```diff
@@ -40,7 +40,7 @@ echo '<?php $rules = array( ' > "$RULE"
 #
 # Looking at statistics like http://stats.jenkins-ci.org/jenkins-stats/svg/201409-jenkins.svg,
 # I think three or four should be sufficient
-for v in 1.554 1.565 1.580 1.596; do
+for v in 1.554 1.565 1.580 1.596 1.609; do
```

Comment:
@orrc Something to think about, but for now we need to get this new version in to fix the site and the stable UC.